cnd/src/config/settings.rs

```diff
@@ -267,4 +269,37 @@ mod tests {
                 listen: vec!["/ip4/0.0.0.0/tcp/9939".parse().unwrap()],
             })
     }
+
+    // Note toml library serializes Network without a trailing new
```

Comment:
Open an issue on toml's library?

Reply:
It seems to be something to with the `Multiaddr` serialization.  When I tried to make a small repro toml crate handled array of ints and and also array of custom type correctly.

Reply:
strange as I would expect the new line **after** the end of the multiaddr array...